### PR TITLE
Feature/civ 3450 updated credential commons

### PR DESCRIPTION
**Pull request by explicit-projects**
Small PR to support async creation of Claims & VerifiableCredentials

Main code change required making these functions sync:
CredentialRequest.acceptClaims
CredentialRequest.createCredential

### CODE REVIEW

File: src/cr/CredentialRequest.js [hunk 1]
```diff
@@ -31,11 +31,11 @@ class CredentialRequest {
     return new CredentialRequest(null, null, R.clone(obj));
   }
 
-  acceptClaims(claims = []) {
-    const claimInstances = claims.map((claim) => {
+  async acceptClaims(claims = []) {
+    const claimInstances = await Promise.all(claims.map(async (claim) => {
```
Comment (dmelosantos) [inline src/cr/CredentialRequest.js:35]: await Promise.all(claims.map(async (claim)

this is really hard to read

File: test/unit/cr/CredentialRequest.test.js [hunk 2]
```diff
@@ -246,6 +258,6 @@ describe('CredentialRequest', () => {
     const credentialObj = JSON.parse(JSON.stringify(crTest.createCredential()));
 
     const shouldFail = crTest.anchorCredential(credentialObj);
-    return expect(shouldFail).to.be.rejectedWith(Error);
+    return expect(shouldFail).to.eventually.be.rejectedWith(Error);
```
Comment (dmelosantos) [inline test/unit/cr/CredentialRequest.test.js:261]: Why eventually?

Comment (explicit-projects) [reply test/unit/cr/CredentialRequest.test.js]: Thought it was needed.... learnt something new. Thanks - I removed it

File: src/cr/CredentialRequest.js (hunk 1 above)
Comment (explicit-projects) [reply src/cr/CredentialRequest.js]: I've split it onto a seperate line

**Review by jpsantosbh — APPROVED** [review]

**Pull request merged**